### ISSUE TITLE
Backend change resets database too late

### ISSUE THREAD

**Issue by andywuest** [andywuest/harbour-watchlist#31]
When changing the backend the new configuration is stored (and database is reset) too late. It is reset after the Watchlist Update is trigged.

**Comment by andywuest** [comment]
The following might work.

```
Page {
  onStatusChanged: {
        if (status === PageStatus.Activating) {
          //
        } else if (status === PageStatus.Deactivating) {
          // store ??
        }
    }
}
```

**Comment by andywuest** [comment]
fixed with 6c6cb4880e02f6eb87b5196cfcc9bc4c011975ee
